src/js/lib/client/src/index.js

```diff
@@ -73,6 +74,11 @@ optParser.options('background-throttling', {
     describe : 'Throttle animations when run in background [false]',
 })
 
+optParser.options('local', {
```

Comment:
I dont think the name `local` is self-explanatory. I would never ever think that `./enso --local ...` means "Set the path of a local project manager to use for running projects"`

Reply:
That was the issue spec. 
What about `--local-project-manager` or `--project-manager`

Reply:
later one!